catalog-rest-service/src/main/resources/json/schema/entity/services/connections/pipeline/dagsterConnection.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/pipeline/dagsterConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "DagsterConnection",
+  "description": "Dagster Metadata Database Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.services.connections.pipeline.DagsterConnection",
+  "definitions": {
+    "DagsterType": {
+      "description": "Service type.",
+      "type": "string",
+      "enum": ["Dagster"],
+      "default": "Dagster"
+    }
+  },
+  "properties": {
+    "type": {
+      "title": "Service Type",
+      "description": "Service Type",
+      "$ref": "#/definitions/DagsterType",
+      "default": "Dagster"
+    },
+    "hostPort": {
+      "title": "Host And Port",
+      "description": "Pipeline Service Management/UI URI.",
+      "type": "string",
+      "format": "uri"
+    },
+    "numberOfStatus": {
+      "description": "Pipeline Service Number Of Status",
+      "type": "integer",
+      "default": "10"
+    },
+    "connection": {
+      "title": "Metadata Database Connection",
+      "description": "Underlying database connection. See https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html for supported backends.",
```

Comment:
I don't think this description is relevant here, we should update it with dagster's docs